hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/EventCondition.java

```diff
@@ -84,9 +99,12 @@ public void setExpression(String expression) {
     }
 
     public boolean match(Event value) {
-        if (null == expression || expression.isEmpty() || null == value) {
+        if (null == value) {
             return false;
         }
+        if (null == expression || expression.isEmpty()) {
+            return true;
+        }
         String[] expressions = expression.split(",");
```

Comment:
I'm not sure I understand your response.  I don't have a problem with complex expressions.  I only meant that it was confusing that "expression" could contain multiple expressions. The terminology seems backwards and it wasn't clear from the jdoc,

Moreover I was concerned about the comma-separated parsing.  But maybe I am not reading it correctly.  In short, will this parse correctly:

event.id start 'ID,YZ', event.context.category == 'Ser,ver', event.tag.from end ',com'